src/collections/programs/ProgramsSingle.style.js

```diff
@@ -1,6 +1,29 @@
 import styled from "styled-components";
```

Comment:
Do not rename this file. No changes are needed in this file. The CSS needs to be added in this file https://github.com/layer5io/layer5/blob/master/src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js